xinetd_init.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# This script configures xinetd
+# Script steps:
+# 1. Install xinetd
+# 2. Copy the example config file
+# 3. Edit the config file
+# 4. Restart xinetd
+# 5. Open ports 443 and 6443
+# 6. Display values for the local machine's /etc/hosts
+
+# Install
+sudo yum install xinetd -y
+# Copy the config file
+if [ ! -f "/etc/xinetd.d/openshift" ]; then
+    sudo cp dev-scripts/openshift_xinetd_example.conf /etc/xinetd.d/openshift
+fi
+# Get the IP values
+addrs=$(cat dev-scripts/ocp/ostest/.openshift_install_state.json | jq '.["*installconfig.InstallConfig"]["config"]["platform"]["baremetal"] | .apiVIP, .ingressVIP')
+apivip=$(echo $addrs | sed '1!d')
+ingressvip=$(echo $addrs | sed '2!d')
+# Replace values
+sed -i "s/<IPv6_API_Address>/${apivip}/g; s/<IPv6_Ingress_Address>/${apivip}/g" /etc/xinetd.d/openshift
+# Restart
+sudo systemctl restart xinetd
```

Comment:
reload would be better, although I am not sure what systemd does in that case if the service is not running yet. You might also need to check that it is started first.

Reply:
`restart` is just a `stop` and `start` combined, but I agree that `reload` is more fitting :)